org.eclipse.winery.yaml.common/src/main/resources/tosca_simple_yaml_non-normative_1_1.yml

```diff
@@ -0,0 +1,168 @@
+tosca_definitions_version: tosca_simple_yaml_1_1
+
+description: >
+  Non-normative type definitions from Chapter 9 of TOSCA Simple Profile in
+  YAML Version 1.1 - Candidate OASIS Standard 01.
+
+#---------------------------------------------------------------------------------------------------------------------#
+#
+# Artifact Types
```

Comment:
That content is already contained in line 12 - I would remove this comment. Also applies for the other comments.

Reply:
Please keep it: Simplifies reading this file!

Reply:
I tried to be consistent with kleinech's format in tosca_simple_yaml_1_1.yaml.

Reply:
If you two insist on keeping this, keep it.

But please do not take this habit into Java code!